### ISSUE TITLE
Foxtrot team planning

### ISSUE THREAD

**Issue by leikkisa** [department-of-veterans-affairs/caseflow#12692]
This is an epic to track Foxtrot's sprint goals on various lines of effort.

[Foxtrot's tickets in ZenHub](https://app.zenhub.com/workspaces/caseflow-5915dd178f67e20b5553ba0c/board?labels=foxtrot&repos=51449239,40141397,52119791,50130088,68742817,130378872,79498890&showPipelineDescriptions=false)

#### Emojis!
✅- Did it!
Please update points completed when adding a checkmark

↪️- Tickets carried over from before

:space_invader: - Extra work pulled in

**Comment by leikkisa** [comment]
# Sprint 11/18 - 11/29
Engineering capacity: 58%

## Asyncable jobs

## Missing ratings

## Multiple legacy opt in issues

## Motions to vacate
1. [Fix action permissions for MTV tasks](https://github.com/department-of-veterans-affairs/caseflow/issues/12691)

**Comment by leikkisa** [comment]
# Sprint 11/18 - 11/29
Engineering capacity: 58%

**Comment by leikkisa** [comment]
# Sprint 11/18 - 11/29
Engineering capacity: 58%

## Sprint work

## Other work

**Comment by leikkisa** [comment]
# Sprint 12/2 - 12/13
Engineering capacity: 100%
Asyncable job duty: Sally (12/2), JC (11/25)

## Sprint status
:arrow_right_hook: Carryover: 
:arrow_right_hook: ✅Carryover completed: 

⭐️ Points pulled in for this sprint: 
⭐️✅Points from this sprint completed: 

👾Space invaders pulled in: 
👾✅Space invaders completed: 

Total points completed: 
Not started and in-progress estimate: 

## Sprint activities

### Asyncable jobs


### Motions to vacate


### Multiple legacy issues


### Accessibility


### Tech improvement


## Other activities

## Demo
Presenter: 
Topic: 

## Retrospective
Retro facilitator:
Retro DJ: Geronimo ;-)

### Retro action items

**Comment by leikkisa** [comment]
# SCORE Sprint 12/16 - 12/27
Engineering capacity: 64%
Batteam: Sally (12/23)
Holiday and winter break

## Sprint status
:arrow_right_hook: Carryover: 18
:arrow_right_hook: ✅Carryover completed: 13
:arrow_right_hook: Remaining: 5

⭐️ Points pulled in for this sprint: 18
⭐️✅Points from this sprint completed: 
⭐️✅Decided not to do: 6
⭐️Remaining: 12

👾Space invaders pulled in: 8
👾✅Space invaders completed: 1
👾Remaining: 7

Total points completed: 14
Remaining: 24
Decided not to do: 6

## Sprint activities

### Asyncable jobs
1. ⭐️(5) Follow up job messaging work #12926

### Motions to vacate
1. ⭐️(3) MTV - Create request issues for vacated decisions #12373
1. ⭐️✅(3) Lit Support: Create a motion to vacate task from case details#12283 - decided this was no longer needed

### Multiple legacy issues
1. :arrow_right_hook: (5) [Create table to connect a request issue to many legacy issues](https://github.com/department-of-veterans-affairs/caseflow/issues/12397)

### Accessibility
1. ⭐️✅(3) under dispatch/Work Assignments the elements Hourly, Daily, Weekly and Monthly should be coded as tabs, and read as tabs by AT #12138

### Tech improvement
1. :arrow_right_hook: ✅(5) [Upgrade our build tooling on the frontend to Webpack 4](https://github.com/department-of-veterans-affairs/caseflow/issues/12414)
1. :arrow_right_hook: ✅(5) [Upgrade to Babel 7+](https://github.com/department-of-veterans-affairs/caseflow/issues/12434)
1. :arrow_right_hook: ✅(3) Convert ui_hash to serializers (decision review files) #12355
1. 👾(2) Update Frontend Dependencies 2019-12 #12996
1. 👾(5) Convert ui_hash to serializers (intake files) #12409

### Sentry alerts
1. ⭐️(3) BGS::ShareError No Records Returned #12166

### Other
1. ⭐️(1) Update copy for legacy issue opt-in selection #12917
1. 👾✅(1) Add missing 930 EP codes #12989

## Demo
Canceled due to holidays

## Retrospective
Canceled due to holidays

### Retro action items

**Comment by leikkisa** [comment]
# Transcontinental Sprint 1/13/2020 - 1/24/2020
Engineering capacity: 80%
Batteam: JC (1/13)
MLK Day (1/20)

## Sprint status
:arrow_right_hook: Carryover: 15
:arrow_right_hook: ✅Carryover completed: 4
:arrow_right_hook: Remaining: 11

⭐️ Points pulled in for this sprint: 22
⭐️✅Points from this sprint completed: 2
⭐️Remaining: 20

👾Space invaders pulled in: 
👾✅Space invaders completed: 
👾Remaining: 

Total to do: 37
Total points completed: 6
Remaining: 31

## Sprint activities

### Asyncable jobs
1. ⭐️(2) Add "Job note with message" method #12947

### Missing ratings
1. ⭐️(3) Allow user to add issues after the receipt date #12289
1. ⭐️(2) ✅Add checkbox for verifying unidentified issue #12380
1. :arrow_right_hook: (3) Handle response for decisions at issue service #13081

### Motions to vacate
1. ⭐️(2) MTV - Add vacated decision issues #13069
1. ⭐️(3) Review vacated decision issues action #13006
1. ⭐️(3) Build decision attorney review screen #13007

### Multiple legacy issues
1. ⭐️(5) Use new table for legacy issue opt ins #12615

### Accessibility
None

### Tech improvement
1. :arrow_right_hook: (5) Convert ui_hash to serializers (intake files) #12409
1. :arrow_right_hook: ✅(1) Deprecate stats pages #12991 #11933
1. :arrow_right_hook: ✅(3) Fix race condition for on_decision_issues_sync_processed #9948
1. :arrow_right_hook: (3) Contested decisions on unidentified issues should go on rating EPs #12952

### Sentry alerts / Batteam
1. ⭐️(2) Do not create decision review task unless there are eligible request issues #11754

### Other


## Demo
Presenter: 
Topic: 

## Retrospective
Facilitator: 
DJ: 

### Retro action items

**Comment by leikkisa** [comment]
# Coast Guard Sprint 1/27/2020 - 2/6/2020
Engineering capacity: 100%

## Sprint status
:arrow_right_hook: Carryover: 16
:arrow_right_hook: ✅Carryover completed: 
:arrow_right_hook: Remaining: 16

⭐️ Points pulled in for this sprint: 24
⭐️✅Points from this sprint completed: 6
⭐️Decided not to do: 
⭐️Remaining: 18

👾Space invaders pulled in: 
👾✅Space invaders completed: 
👾Remaining: 0

Total to do: 41
Total points completed: 19
Decided not to do: 6
Remaining: 16

## Sprint activities

### Asyncable jobs
1. ⭐️✅(2) Add "Job note with message" method #12947

### Ability to intake more decision types
1. ⭐️(5) Difference of Opinion | Handle dispositions #10815

### Missing ratings
1. :arrow_right_hook: (3) Handle response for decisions at issue service #13081
1. ⭐️(3) Handle save for verified unidentified issue #12382

### Motions to vacate
1. ⭐️✅(2) Motions to Vacate | Motion Attorney | UAT Fix #13003
1. ⭐️✅(1) Motion to Vacate | VLJ | Address Motion | Display Return to Motion Attorney Option For Dismiss #13243
1. ⭐️✅(1) Motion to Vacate | Denied | Remove URL Field Requirement Condition #13236
1. ⭐️(3) Motion to Vacate | Support multiple AMA appeal streams #13211

### Multiple legacy issues
1. :arrow_right_hook: (5) Use new table for legacy issue opt ins #12615

### Accessibility
1. ⭐️(2) Claims folder details element should be read as a button with expanding/collapsing functionality #12143

### Tech improvement
1. :arrow_right_hook: (5) Convert ui_hash to serializers (intake files) #12409
1. :arrow_right_hook: (3) Contested decisions on unidentified issues should go on rating EPs #12952
1. ⭐️(3) Create Draft Pull Request for Controller-Level Schema Validations #13181

### Sentry alerts / Batteam
1. ⭐️(2) new exception class for no end products returned #11342

### Other
1. 👾BVA Intake | No BVA Policy Regarding Same RO or Station | VBA CorpDB Rule To Be Removed #13165 - note, this is done but waiting to merge

**Comment by leikkisa** [comment]
# Friendship 7 Sprint 2/10/2020 - 2/21/2020
Engineering capacity: 

## Sprint status
:arrow_right_hook: Carryover: 
:arrow_right_hook: ✅Carryover completed: 
:arrow_right_hook: Remaining: 

⭐️ Points pulled in for this sprint: 
⭐️✅Points from this sprint completed: 
⭐️Decided not to do: 
⭐️Remaining: 

👾Space invaders pulled in: 
👾✅Space invaders completed: 
👾Remaining: 

Total to do: 
Total points completed: 
Decided not to do: 
Remaining: 

## Sprint activities

### Asyncable jobs

### Ability to intake more decision types
1. :arrow_right_hook: (5) Difference of Opinion | Handle dispositions #10815 (completed, awaiting review)

### Missing ratings
1. :arrow_right_hook: (3) Handle response for decisions at issue service #13081

### Motions to vacate
1. ⭐️(3) Motion to Vacate | Associate PostDecisionMotion to Appeal #13355
1. ⭐️(3) Motion to Vacate | Granted | Update attorney review screen #13007
1. ⭐️(3) Motion to Vacate | Granted | Readjudicate | Decision Attorney | Go to Add Decisions screen #13071
1. ⭐️(3) Motion to Vacate | Create "De Novo" appeal stream #13336

### Multiple legacy issues
1. :arrow_right_hook: (5) Use new table for legacy issue opt ins #12615

### Accessibility
1. :arrow_right_hook: (2) Claims folder details element should be read as a button with expanding/collapsing functionality #12143

### Tech improvement
1. :arrow_right_hook: (5) Convert ui_hash to serializers (intake files) #12409
1. :arrow_right_hook: (3) Create Draft Pull Request for Controller-Level Schema Validations #13181

### Sentry alerts / Batteam
1. ⭐️(2) Clean up new lines in rating issue description before saving #13311

### Other

**Comment by leikkisa** [comment]
# Peace Corps Sprint 2/24/2020 - 3/9/2020
Engineering capacity: 63%

## Sprint status
:arrow_right_hook: Carryover: 23
:arrow_right_hook: ✅Carryover completed: 
:arrow_right_hook: Remaining: 23

⭐️ Points pulled in for this sprint: 14
⭐️✅Points from this sprint completed: 
⭐️Decided not to do: 
⭐️Remaining: 24

👾Space invaders pulled in: 
👾✅Space invaders completed: 
👾Remaining: 

Total to do: 
Total points completed: 
Decided not to do: 
Remaining: 

## Sprint activities

### Asyncable jobs

### Ability to intake more decision types
None

### Missing ratings
1. :arrow_right_hook: (3) Handle response for decisions at issue service #13081

### Motions to vacate
1.  :arrow_right_hook: (3) Motion to Vacate | Create "De Novo" appeal stream #13336

### Multiple legacy issues
1. :arrow_right_hook: (5) Use new table for legacy issue opt ins #12615

### Accessibility
None

### Tech improvement
1. :arrow_right_hook: (5) Convert ui_hash to serializers (intake files) #12409
1. :arrow_right_hook: (3) Create Draft Pull Request for Controller-Level Schema Validations #13181

### Sentry alerts / Batteam
1. ⭐️(2) Clean up new lines in rating issue description before saving #13311

### Other

**Comment by leikkisa** [comment]
# TBD Sprint 3/9/2020 - 3/20/2020
Engineering capacity: 88%
- JC Batteam week of 3/9

## Sprint status
:arrow_right_hook: Carryover: 20
:arrow_right_hook: ✅Carryover completed:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 27
⭐️✅Points from this sprint completed:
⭐️Decided not to do:
⭐️Remaining:

👾Space invaders pulled in:
👾✅Space invaders completed:
👾Remaining:

Total to do: 47
Total points completed:
Decided not to do:
Remaining:

## Sprint activities

### Intake
1. :arrow_right_hook: (3) Intake | Restrict Non-BVA MailTeam Members From Starting Board Appeals #13356
1. ⭐️ (3) Intake | Add Contention service to ruby-bgs #13488
1. ⭐️ (3) Intake | Use user station for station of jurisdiction during claim establishment #13247

### Missing ratings
1. :arrow_right_hook: (3) Handle response for decisions at issue service #13081

### Motions to vacate
1. :arrow_right_hook: (3) Motion to Vacate | Return to Judge front-end #13474
1. ⭐️ (2) Motion to Vacate | Create only 1 appeal stream after judge addresses motion #13617
1. ⭐️ (3) Motion to Vacate | Fix Link to Chairman Memorandum and all assets #13611
1. ⭐️ (3) Motion to Vacate | Fix success banner for "Created Motion to Vacate" #13615
1. ⭐️ (3) Motion to Vacate | Code audit #13507
1. ⭐️ (3) Motion to Vacate | Frontend clean-up #13610
1. ⭐️ (3) Motion to Vacate | Return to Judge (back end) #13475
1. ⭐️ (2) Motion to Vacate | Hide Motion to Vacate task option until appeal is outcoded #13614
1. ⭐️ (2) Motion to Vacate | Success banner after motions attorney sends recommendation to judge #13622

### Multiple legacy issues
1. :arrow_right_hook: (5) Use new table for legacy issue opt ins #12615

### Accessibility
None

### Tech improvement
1. :arrow_right_hook: (5) Convert ui_hash to serializers (intake files) #12409
1. :arrow_right_hook: (1) Allow nullish coalescing operator in eslint #13426

### Sentry alerts / Batteam
None

### Other

**Comment by leikkisa** [comment]
# Sandra Day O’Connor Sprint 3/23/2020 - 4/3/2020
Engineering capacity: 88%
- Sandra Batteam week of 3/23

## Sprint status
:arrow_right_hook: Carryover: 23
:arrow_right_hook: ✅Carryover completed: 3
:arrow_right_hook: Remaining: 20
:arrow_right_hook: 🛑 (blocked)

⭐️ Points pulled in for this sprint: 17
⭐️✅Points from this sprint completed:
⭐️Decided not to do:
⭐️Remaining:

👾Space invaders pulled in:
👾✅Space invaders completed:
👾Remaining:

Total to do: 40
Total points completed: 3
Decided not to do:
Remaining:  37

## Sprint activities

### Intake
1. :arrow_right_hook: 🛑 (3) Intake | Restrict Non-BVA MailTeam Members From Starting Board Appeals #13356
1. :arrow_right_hook: ✅ (3) Intake | Validate veteran address #13393
1. ⭐️ (5) Intake | Do not allowing removing contentions with exam scheduled #13738
1. ⭐️ (3) Intake | ITF | Support claim label for SCs with issues decided > 1 year ago #13647
1. ⭐️ (2) Intake | Validate veteran name_suffix #13391
1. ⭐️ (2) Bug | Fix asynchronous request issue withdrawals in VBMS #13483
1. ⭐️ (2) Upstream error fix | Validate that birthdates values have leading zeros #13589
1. ⭐️ (3) Intake | Validate claimant address #13394


### Missing ratings
1. :arrow_right_hook: (3) Handle response for decisions at issue service #13081

### Motions to vacate
1. :arrow_right_hook: (3) Motion to Vacate | Fix Link to Chairman Memorandum and all assets #13611


### Multiple legacy issues
1. :arrow_right_hook: (5) Use new table for legacy issue opt ins #12615

### Accessibility
None

### Tech improvement
1. :arrow_right_hook: (5) Convert ui_hash to serializers (intake files) #12409
1. :arrow_right_hook: (1) Allow nullish coalescing operator in eslint #13426

### Sentry alerts / Batteam
None

### Other

**Comment by leikkisa** [comment]
# Civil Rights Sprint Sprint 4/6/2020 - 4/17/2020
Engineering capacity: 88%
- Sally Batteam week of 4/12

## Sprint status
:arrow_right_hook: Carryover: 13
:arrow_right_hook: ✅Carryover completed: 5
:arrow_right_hook: Remaining: 8

⭐️ Points pulled in for this sprint: 17
⭐️✅Points from this sprint completed: 0
⭐️🛑Decided not to do: 
⭐️Remaining: 17

👾Space invaders pulled in:
👾✅Space invaders completed:
👾Remaining:

Total to do: 30
Total points completed: 5
Decided not to do:
Remaining:  25

## Sprint activities
Note: We have started putting on hold work back into ready for dev in order to reconsider their priority, so the "Carryover" points has been reduced.

### Intake


### COVID-19 Update
1. ⭐️ (3) Intake | COVID 19 Update | Show untimely issue modal after legacy issue selection #13857
1. ⭐️ (5) Intake | COVID-19 Update | Allow timeliness exemptions for legacy issue opt-ins #13817
1. ⭐️ (3) Intake | COVID-19 update | Update eligibility rules #13858

### Missing ratings
1. :arrow_right_hook: ✅ (5) Handle response for decisions at issue service #13081

### Motions to vacate
1. :arrow_right_hook: (3) Motion to Vacate | Fix Link to Chairman Memorandum and all assets #13611
1. ⭐️ (2) Motion to Vacate | Allow partial grant option for Motions Attorneys #13864
1. ⭐️🛑 (2) Motion to Vacate | Include draft hyperlink for all vacate types #13861
1. ⭐️🛑 (2) Motion to Vacate | Update Return to Motions Attorney #13862

### Multiple legacy issues
Moved back to ready for dev

### Accessibility
None

### Tech improvement
1. :arrow_right_hook: (5) Convert ui_hash to serializers (intake files) #12409
1. :arrow_right_hook: Investigate Form-Management Libraries #13257
1. 👾 Fixed Routing for Organizations Views #13903

### Sentry alerts / Batteam
None

### Other

**Comment by leikkisa** [comment]
# Sprint 4/20/2020 - 5/1/2020
Engineering capacity: 75%
- JC Batteam week of 4/20
- Sandra Batteam week of 4/27

## Sprint status
:arrow_right_hook: Carryover: 7
:arrow_right_hook: ✅Carryover completed:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 22
⭐️✅Points from this sprint completed:
⭐️🚫Decided not to do:
⭐️Remaining:
⭐️🛑Blocked for now:

👾Space invaders pulled in:
👾✅Space invaders completed:
👾Remaining:

Total to do: 29
Total points completed:
Decided not to do:
Remaining:  

## Sprint activities
Note: We have started putting on hold work back into ready for dev in order to reconsider their priority, so the "Carryover" points has been reduced.

### Intake
1. ⭐️(5) Intake | Do not allowing removing contentions with exam scheduled #13738

### COVID-19 Update
1. :arrow_right_hook: (In review) (3) Intake | COVID-19 update | Update eligibility rules #13858
1. :arrow_right_hook: (In review) (1) Intake | COVID-19 Update | Update NOD date rules #13901

### Missing ratings
1. ⭐️(3) Missing Ratings | Assessment: RatingWebService.findRatingData among current Caseflow BGS services in production #13996

### Motions to vacate
1. :arrow_right_hook: (3) Motion to Vacate | Fix Link to Chairman Memorandum and all assets #13611
1. ⭐️(2) Motion to Vacate | Add Pulac-Cerullo task action to Judge Task #13920
1. ⭐️(3) Motion to Vacate | Admin Actions in De Novo flow #13918

### Attorney fees
1. ⭐️(2) Contested Claims | Attorney Fees | Create list of existing attorney-fee cases #14002
1. ⭐️(3) Contested Claims | Attorney Fees | Evaluate coverage of attorneys in external systems #14004

### Multiple legacy issues
None

### Accessibility
None

### Tech improvement
1. :arrow_right_hook: Investigate Form-Management Libraries #13257

### Sentry alerts / Batteam / Bugs
1. ⭐️(2) TypeError: Cannot read property 'description' of undefined in IntakeEdit. #13232
1. ⭐️(2) Bug | Fix asynchronous request issue withdrawals in VBMS #13483

### Other

**Comment by leikkisa** [comment]
Sprint 5/4/2020 - 5/15/2020

**Comment by leikkisa** [comment]
Sprint 5/8/2020 - 5/29/2020

**Comment by leikkisa** [comment]
# March Against Fear Sprint Sprint 6/1/2020 - 6/12/2020
Engineering capacity: 73%
- Yang PTO 6/1
- JC Batteam week of 6/1
- Sandra Batteam week of 6/8

## Sprint status
:arrow_right_hook: Carryover: 11
:arrow_right_hook: ✅Carryover completed:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 17
⭐️✅Points from this sprint completed:
⭐️🚫Decided not to do:
⭐️Remaining:
⭐️🛑Blocked for now:

👾Space invaders pulled in: 1
👾✅Space invaders completed:
👾Remaining:

Total to do:
Total points completed:
Decided not to do:
Remaining:  

## Sprint activities

### Intake
1. :arrow_right_hook: (3) Intake | Prevent removing an issue if connected to a rating #13977 (pulled in last sprint as an extra)
1. ⭐️ (2) Intake | Upstream Error | Add error classes (and ignore in Sentry) #14417

### Motions to vacate
- Pilot has started

### Attorney fees
1. :arrow_right_hook: (3) Contested Claims | Attorney Fees | Fetch Attorney Fee Data #14294 (broken out from ticket from last sprint)
1. ⭐️ (2) Contested Claims | Attorney Fees | Unlisted claimant storage #14380
1. ⭐️ (3) Contested Claims | Attorney Fees | Add ‘Claimant not listed’ selection when attorney is not available #14107
1. 👾 (1) Caseflow Intake | Bug | Fix Debounce for Add Claimant Search #14430

### Legacy opt-ins
1. :arrow_right_hook: (3) BVA | Intake | Legacy opt-in | Extend eligibility period that expires during a federal holiday #14285

### Accessibility
1. ⭐️ (3) Intake | The 'enter or select form box' and 'Which form' label is not read by assistive technology (End: 6/12) #14373
1. ⭐️ (2) Dispatch: Special Issues checkboxes are not grouped together (End: 7/10) #12123
1. ⭐️ (2) Demo | Assistive technology does not read the access link for each product in demo (End: 6/12) #14377

### Tech improvement
1. :arrow_right_hook: Investigate Form-Management Libraries #13257
1. :arrow_right_hook: Handling of PDF Files via Webpack #13917
1. :arrow_right_hook: (2) Add one controller schema for validation and documentation #14017
1. ⭐️ (3) Build out Stories for Remaining Common UI Components #14021

**Comment by leikkisa** [comment]
# Stonewall Sprint Sprint 6/15/2020 - 6/26/2020
Engineering capacity: 83%
- Yang batteam week of 6/15
- Nava holiday 6/19 (JC, Sally, Yang)

## Sprint status
:arrow_right_hook: Carryover: 15
:arrow_right_hook: ✅Carryover completed:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 11
⭐️✅Points from this sprint completed:
⭐️🚫Decided not to do:
⭐️🛑Blocked for now:
⭐️Remaining:

👾Space invaders pulled in:
👾✅Space invaders completed:
👾Remaining:

## Sprint activities

### Intake
1. :arrow_right_hook: (2) Intake | Upstream Error | Add error classes (and ignore in Sentry) #14417

### Attorney fees
1. :arrow_right_hook: (3) Contested Claims | Attorney Fees | Add ‘Claimant not listed’ selection when attorney is not available #14107
1. ⭐️ (3) Contested Claims | Attorney Fees | Refactor Claimant model into subclasses #14502
1. ⭐️ (2) Contested Claims | Attorney Fees | Add new Claimant subclasses #14503

### Accessibility
1. ⭐️ (3) (note: this is to unblock #14373) Update `react-select` to Latest #14490
1. :arrow_right_hook: (3) Intake | The 'enter or select form box' and 'Which form' label is not read by assistive technology (End: 6/12) #14373
1. :arrow_right_hook: (2) Dispatch: Special Issues checkboxes are not grouped together (End: 7/10) #12123
1. :arrow_right_hook: (2) Demo | Assistive technology does not read the access link for each product in demo (End: 6/12) #14377
1. ⭐️ (3) Filter button functionality not accessible using JAWS keyboard navigation (End: 7/10) #12902

### Tech improvement
1. :arrow_right_hook: Investigate Form-Management Libraries #13257
1. :arrow_right_hook: Handling of PDF Files via Webpack #13917
1. :arrow_right_hook: (3) Build out Stories for Remaining Common UI Components #14021

**Comment by leikkisa** [comment]
# Thurgood Marshall Sprint Sprint 6/29/2020 - 7/10/2020
Engineering capacity: 80%
- Sally batteam week of 7/6
- Independence day 7/3 (JC, Sally, Yang)

## Sprint status
:arrow_right_hook: Carryover: 14
:arrow_right_hook: ✅Carryover completed:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 20
⭐️✅Points from this sprint completed:
⭐️🚫Decided not to do:
⭐️🛑Blocked for now:
⭐️Remaining:

👾Space invaders pulled in:
👾✅Space invaders completed:
👾Remaining:

## Sprint activities

### Attorney fees
1. :arrow_right_hook: (3) Contested Claims | Attorney Fees | Refactor Claimant model into subclasses #14502
1. :arrow_right_hook: (2) Contested Claims | Attorney Fees | Add new Claimant subclasses #14503
1. ⭐️ (3) Contested Claims | Attorney Fees | Update intake review endpoint #14505

### Accessibility
1. :arrow_right_hook: (3) (note: this is to unblock #14373) Update `react-select` to Latest #14490
1. :arrow_right_hook: (3) Intake | The 'enter or select form box' and 'Which form' label is not read by assistive technology (End: 6/12) #14373

### G code disposition
1. ⭐️ (2) Intake | G Code | Store original legacy appeal data in case of rollback #14500
1. ⭐️ (3) Intake | G Code | Backfill original legacy appeal data #14575
1. ⭐️ (3) Intake | G Code | Add ability to process "G" code opt-ins #14574
1. ⭐️ (2) Intake | G Code | Add ability to process "G" code rollbacks #14619

### Intake - other
1. ⭐️ (estimate TBD) Intake | Bug | No legacy appeal found for issue #14592
1. ⭐️ (2) Business Line | Display task creator name CSS ID date time in form #13067
1. ⭐️ (3) Business Lines | Report | Export non-comp LOB tasks CSV #13075
1. ⭐️ (2) Caseflow Intake: Review page spacing consistency #14488

### Tech improvement
1. :arrow_right_hook: Investigate Form-Management Libraries #13257
1. :arrow_right_hook: Handling of PDF Files via Webpack #13917
1. :arrow_right_hook: (3) Build out Stories for Remaining Common UI Components #14021

**Comment by leikkisa** [comment]
# Sprint 7/13/2020 - 7/24/2020
Engineering capacity: 63%
- JC batteam week of 7/13
- Sandra batteam week of 7/20
- JC at reduced capacity week of 7/20

## Sprint status
:arrow_right_hook: Carryover: 15
:arrow_right_hook: ✅Carryover completed: 3
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 11
⭐️✅Points from this sprint completed:
⭐️🚫Decided not to do:
⭐️🛑Blocked for now:
⭐️Remaining:

👾Space invaders pulled in:
👾✅Space invaders completed:
👾Remaining:

## Sprint activities

### Attorney fees
1. :arrow_right_hook: (2) Contested Claims | Attorney Fees | Add new Claimant subclasses #14503
1. :arrow_right_hook: (3) Contested Claims | Attorney Fees | Update intake review endpoint #14505

### G code disposition
1. :arrow_right_hook: (3) Intake | G Code | Backfill original legacy appeal data #14575
1. :arrow_right_hook: (3) Intake | G Code | Add ability to process "G" code opt-ins #14574
1. :arrow_right_hook: (2) Intake | G Code | Add ability to process "G" code rollbacks #14619

### Missing ratings
1. ⭐️ (3) Intake | Missing Ratings | Handle unpromulgated ratings from RatingAtIssue #14654
1. ⭐️ (2) Intake | Missing Ratings | Add service to fetch current rating profile #14653

### Motions to vacate
1. ⭐️ (3) Motions to Vacate | Create internal report #14605

### Intake - other
1. :arrow_right_hook: (estimate TBD) Intake | Bug | No legacy appeal found for issue #14592
1. :arrow_right_hook:✅ (3) Business Lines | Report | Export non-comp LOB tasks CSV #13075
1. :arrow_right_hook: (2) Caseflow Intake: Review page spacing consistency #14488

### Tech improvement
1. ⭐️ (3) Upstream Error | investigate failed DecisionDocument jobs #14079

### Not active work - excluding from point counts
1. :arrow_right_hook: Investigate Form-Management Libraries #13257
1. :arrow_right_hook: Handling of PDF Files via Webpack #13917
1. :arrow_right_hook: (3) Build out Stories for Remaining Common UI Components #14021

**Comment by leikkisa** [comment]
# Voting Rights Sprint 7/27/2020 - 8/7/2020
Engineering capacity:75%
- Yang batteam week of 7/27
- Sandra OOO 7/27-7/30
- Sally PTO 7/31

## Sprint status
:arrow_right_hook: Carryover: 13
:arrow_right_hook: ✅Carryover completed: 3
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 16
⭐️✅Points from this sprint completed:
⭐️🚫Decided not to do:
⭐️🛑Blocked for now:
⭐️Remaining:

👾Space invaders pulled in:
👾✅Space invaders completed:
👾Remaining:

## Sprint activities

## Attorney fees
1. :arrow_right_hook:✅ (3) Contested Claims | Attorney Fees | Update intake review endpoint #14505

### Change docket type
1. ⭐️ (5) Ability to change docket type | Prelim engineering investigation #14657

### Edit EP claim labels
1. ⭐️ (5) Edit EP Claim Labels | Preliminary Engineering investigation #14737

### G code disposition
1. :arrow_right_hook: (2) Intake | G Code | Add ability to process "G" code rollbacks #14619

### Motions to vacate
1. :arrow_right_hook: (3) Motions to Vacate | Create internal report #14605

### Intake - other
1. :arrow_right_hook: (estimate TBD) Intake | Bug | No legacy appeal found for issue #14592
1. :arrow_right_hook: (2) Caseflow Intake: Review page spacing consistency #14488
1. ⭐️ (3) Intake | Prevent removing an issue if connected to a rating #13977

### Upstream errors
1. :arrow_right_hook: (3) Upstream Error | investigate failed DecisionDocument jobs #14079
1. ⭐️ (3) Intake | Upstream Errors | Validate postal code #11942

### Not active work - excluding from point counts
1. :arrow_right_hook: Investigate Form-Management Libraries #13257
1. :arrow_right_hook: Handling of PDF Files via Webpack #13917
1. :arrow_right_hook: (3) Build out Stories for Remaining Common UI Components #14021

**Comment by leikkisa** [comment]
# Social Security Act Sprint Sprint 8/10/2020 - 8/21/2020
Engineering capacity:88%
- Sally batteam week of 8/17

## Sprint status
:arrow_right_hook: Carryover: 8
:arrow_right_hook: ✅Carryover completed:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 12
⭐️✅Points from this sprint completed: 1
⭐️🚫Decided not to do:
⭐️🛑Blocked for now:
⭐️Remaining: 11

👾Space invaders pulled in:
👾✅Space invaders completed:
👾Remaining:

## Sprint activities

## Attorney fees
1. ⭐️ (1) Attorney Fees | Remove one non-rating issue category #14904

### Non-veteran claimants
1. ⭐️ (5) Non-veteran claimants | Prelim engineering investigatio #14736

### Change docket type


### Edit EP claim labels


### G code disposition


### Motions to vacate


### Intake - other
1. ⭐️ (5) New Fiduciary 030 and 040 EP and claim labels to be added to Caseflow Intake #14789
1. ⭐️ (1) ✅ Intake | Add reminder to use encrypted email for PII when adding claimants #10586
1. 👾 (2) Display legacy opt-in selection on "Add Issues" page #14698
1. :arrow_right_hook: (estimate TBD) Intake | Bug | No legacy appeal found for issue #14592
1. :arrow_right_hook: (2) Caseflow Intake: Review page spacing consistency #14488
1. :arrow_right_hook: (3) Intake | Prevent removing an issue if connected to a rating #13977

### Upstream errors
1. :arrow_right_hook: (3) Intake | Upstream Errors | Validate postal code #11942

### Not active work - excluding from point counts
1. :arrow_right_hook: Investigate Form-Management Libraries #13257
1. :arrow_right_hook: Handling of PDF Files via Webpack #13917

**Comment by leikkisa** [comment]
# Edith Sampson Sprint Sprint 8/24/2020 - 9/4/2020
Engineering capacity:63%
- JC batteam week of 8/24
- Yang PTO week of 8/24
- Sandra batteam week of 8/31

## Sprint status
:arrow_right_hook: Carryover: 6
:arrow_right_hook: ✅ Carryover completed:
:arrow_right_hook: 🛑 Blocked for now:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 13
⭐️✅ Points from this sprint completed:
⭐️🚫 Decided not to do:
⭐️🛑 Blocked for now:
⭐️ Remaining:

👾 Space invaders pulled in:
👾✅ Space invaders completed:
👾 Remaining:

## Sprint activities

## Attorney fees
1. ⭐️ (1) Attorney Fees | Capitalize Veteran in claimant relationship field #14946
1. ⭐️ (1) Attorney Fees | Add Clearable prop to searchable dropdown #14998

### Intake
1. :arrow_right_hook: (3) Intake | Prevent removing an issue if connected to a rating #13977
1. ⭐️ (5) Missing Ratings | Add current rating profile endpoint to Caseflow #14839
1. ⭐️ (1) Business Line | Update Copy For Completed Tab #13106
1. ⭐️ (2) Incorrect default text field value when editing issue description #14994
1. ⭐️ (3) Bug | Fix code for cancelling an EP in BGS #14912

### Upstream errors
1. :arrow_right_hook: (3) Upstream Error | Add pay grade validation matching VBMS #14338

**Comment by leikkisa** [comment]
# Sprint 9/7/2020 - 9/18/2020
Engineering capacity: 80%
- Yang batteam week of 9/7
- Labor day for all on 9/7

## Sprint status
:arrow_right_hook: Carryover: 4
:arrow_right_hook: ✅ Carryover completed:
:arrow_right_hook: 🛑 Blocked for now:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 15
⭐️✅ Points from this sprint completed:
⭐️🚫 Decided not to do:
⭐️🛑 Blocked for now:
⭐️ Remaining:

👾 Space invaders pulled in:
👾✅ Space invaders completed:
👾 Remaining:

## Sprint activities

### Attorney fees
1. :arrow_right_hook: (1) Attorney Fees | Add Clearable prop to searchable dropdown #14998

### Intake
1. ⭐️ (2) Caseflow Intake: Review page spacing consistency #14488

### Upstream errors
1. :arrow_right_hook: (3) Upstream Error | Add pay grade validation matching VBMS #14338
1. ⭐️ (2) Upstream Error | Do not sync rating issue data for nonrating board grant effectuations #15019


### Docket type
1. ⭐️ (2) Intake | Docket Change | Eng | Create ClerkOfTheBoard Organization #15135
1. ⭐️ (3) Docket Change | Eng | Create DocketSwitchMailTask #15136

### Edit EP Claim Labels
1. ⭐️ (3) Edit EP Claim Labels | Add UpdateBenefitClaim endpoint to ruby-bgs #14888
1. ⭐️ (3) Edit EP Claim Labels | Add map of end product code characteristics #15114

**Comment by leikkisa** [comment]
# The Emancipation Proclamation Sprint 9/21/2020 - 10/2/2020
Engineering capacity: 73%
- Sally batteam week of 9/28
- JC capacity reduced by 10% week of 9/21, 20% week of 9/28

## Sprint status
:arrow_right_hook: Carryover:
:arrow_right_hook: ✅ Carryover completed:
:arrow_right_hook: 🛑 Blocked for now:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 15
⭐️✅ Points from this sprint completed:
⭐️🚫 Decided not to do:
⭐️🛑 Blocked for now:
⭐️ Remaining:

👾 Space invaders pulled in: 4
👾✅ Space invaders completed:
👾 Remaining:

## Sprint activities

### Attorney fees #13505
1. 👾 (2) Attorney Fees | Bug Fix | Allow adding attorney claimant when veteran has no relationships #15200

### Intake


### Upstream errors #13338


### Docket type #13978
1. ⭐️ (2) Docket Change | Eng | Add Model for `DocketChange` #15137
1. ⭐️ (3) Intake | Docket Change | Eng | Create JudgeAddressDocketSwitchTask #15255
1. 👾 (2) Intake | Docket Change | Task success banner #15094

### Edit EP Claim Labels #11490
1. ⭐️ (3) Edit EP Claim Labels | Add "type" to request issue #15236

### Other
1. ⭐️ (3) Investigate missing remand reason error when adding to MTV stream #15224
1. ⭐️ (3) Intake | Something went wrong - intake not completing properly on submit #15190

**Comment by Theresa1976** [comment]
# The Eleanor Roosevelt Sprint 10/6/2020- 10/16/2020
Engineering capacity: 47%
- Sally batteam week of 10/5-10/9
- Sandra batteam week of 10/12-10/16
- JC capacity reduced by 10% week of 10/5, 100% week of 10/12

## Sprint status
:arrow_right_hook: Carryover:
:arrow_right_hook: ✅ Carryover completed:
:arrow_right_hook: 🛑 Blocked for now:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 14
⭐️✅ Points from this sprint completed:
⭐️🚫 Decided not to do:
⭐️🛑 Blocked for now:
⭐️ Remaining:

👾 Space invaders pulled in: 
👾✅ Space invaders completed:
👾 Remaining:

## Sprint activities

### Attorney fees #13505
1. 👾 (2) Attorney Fees | Bug Fix | Allow adding attorney claimant when veteran has no relationships #15200

### Intake


### Upstream errors #13338


### Docket type #13978
1. ⭐️ (2) Docket Change | Eng | Add Model for `DocketChange` #15137
1. ⭐️ (3) Intake | Docket Change | Eng | Create JudgeAddressDocketSwitchTask #15255
1. 👾 (2) Intake | Docket Change | Task success banner #15094

### Edit EP Claim Labels #11490
1. ⭐️ (3) Edit EP Claim Labels | Add "type" to request issue #15236

### Other
1. ⭐️ (3) Investigate missing remand reason error when adding to MTV stream #15224
1. ⭐️ (3) Intake | Something went wrong - intake not completing properly on submit #15190

**Comment by Theresa1976** [comment]
# The Dizzy Gillespie Sprint  10/19/2020- 10/30/2020 **(Updating in Progress)**
Engineering capacity: 66%
- Yang capacity reduced by 10.5% (week of 10/19-Feature leading and SAFe training)
- Yang capacity reduced by 10% (week of 10/26- Feature leading)
- JC capacity reduced to 100% (week of 10/26 - Other work)

## Sprint status
:arrow_right_hook: Carryover:
:arrow_right_hook: ✅ Carryover completed:
:arrow_right_hook: 🛑 Blocked for now:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 14
⭐️✅ Points from this sprint completed:
⭐️🚫 Decided not to do:
⭐️🛑 Blocked for now:
⭐️ Remaining:

👾 Space invaders pulled in: 
👾✅ Space invaders completed:
👾 Remaining:

## Sprint activities

### Attorney fees #13505
1. ⭐️ (2) Attorney Fees | Deployment #15367 

### Intake



### Upstream errors #13338
1. ⭐️ (2) Intake | Upstream Error | Skip syncing not found end products on intake #15296 

### Docket type #13978
1. ⭐️ (2) Docket Change | Eng | Add Model for `DocketChange` #15137
1. :arrow_right_hook:  (3) Intake | Docket Change | Eng | Create JudgeAddressDocketSwitchTask #15255 
1. ⭐️ (5) Intake | Docket Change | Docket switch recommendation #15331 

### Edit EP Claim Labels #11490
1. ⭐️(3) Edit EP Claim Labels | Add End Product Update model #15330 
1. ⭐️(2) Edit EP Claim Labels | Show EP data on search page #15110

**Comment by Theresa1976** [comment]
# The Dizzy Gillespie Sprint  10/19/2020- 10/30/2020 
Engineering capacity: 66%
- Yang capacity reduced by 10.5% (week of 10/19-Feature leading and SAFe training)
- Yang capacity reduced by 10% (week of 10/26- Feature leading)
- JC capacity reduced to 100% (week of 10/26 - Other work)

## Sprint status
:arrow_right_hook: Carryover:
:arrow_right_hook: ✅ Carryover completed:
:arrow_right_hook: 🛑 Blocked for now:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 14
⭐️✅ Points from this sprint completed:
⭐️🚫 Decided not to do:
⭐️🛑 Blocked for now:
⭐️ Remaining:

👾 Space invaders pulled in: 
👾✅ Space invaders completed:
👾 Remaining:

## Sprint activities

### Motion to Vacate #15224 
1. ⭐️ (5) Motions to Vacate | Add ability to add remand reasons for issues that are remanded #15425 

### Intake Docket Switch #13978 
1. ⭐️(2) Docket Switch | Create Docket switch decision tasks #15424 
1. :arrow_right_hook: Carryover: (2) Docket Switch | Implement Text Formatting & API Post in RecommendDocketSwitchContainer #15420 




### Intake Bug Investigation
1. ⭐️ (2) Intake | Bug investigation - cancel tasks when all issues are removed or ineligible #13575 

### Docket type #13978
1. ⭐️ (2) Intake | Docket Change | Recommendation success banner #15333 

### Edit EP Claim Labels #11490
1. ⭐️(3) Edit EP Claim Labels | Show End Product Type on "Correct issues" page #15109 
1.  ⭐️(3) Edit EP Claim Labels | Handle updating correction type #15128

**Comment by Theresa1976** [comment]
# Veterans Day Sprint  (11/2/2020- 11/13/2020)
Engineering capacity: 67%
Committed to 12 points
- Sandra capacity reduced by 50% - Batteam week of 11/2/2020
- Sally capacity reduced by 50% - Batteam week of 11/9/2020
- Yang capacity reduced by 10% - PTO 11/6/2020
- Yang capacity reduced by 0.5% - Feature Leading 11/2/2020 
- Yang capacity reduced by 0.5% - Feature Leading 11/9/2020

## Sprint status
:arrow_right_hook: Carryover:
:arrow_right_hook: ✅ Carryover completed:
:arrow_right_hook: 🛑 Blocked for now:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 14
⭐️✅ Points from this sprint completed:
⭐️🚫 Decided not to do:
⭐️🛑 Blocked for now:
⭐️ Remaining:

👾 Space invaders pulled in: 
👾✅ Space invaders completed:
👾 Remaining:

## Sprint activities

### Docket type #13978
1. ⭐️ (2) Intake| Docket Change | Receive VLJ ruling #15338  (Issue is Blocked)
1. ⭐️ (3) Intake| Docket Change| VLJ ruling view #15336
1. ⭐️ (2) Intake| Docket Change| Recommendation success banner #15333 

### Edit EP Claim Labels #11490
1. ⭐️(2) Edit EP Claim Labels | Non-rating to rating updates #15126
1. ⭐️(3) Edit EP Claim Labels | Ability to select new claim label #15112

**Comment by Theresa1976** [comment]
# Black Beard the Pirate Sprint  (11/16/2020- 11/27/2020) - In Progress
Engineering capacity: 
Committed to  points
- Sandra capacity reduced by 50% - Batteam week of 11/2/2020
- Sally capacity reduced by 50% - Batteam week of 11/9/2020
- Yang capacity reduced by 10% - PTO 11/6/2020
- Yang capacity reduced by 0.5% - Feature Leading 11/2/2020 
- Yang capacity reduced by 0.5% - Feature Leading 11/9/2020
- Rutvi capacity reduced by 10.5% - SAFe Training 11/4/2020- 11/6/2020

## Sprint status
:arrow_right_hook: Carryover:
:arrow_right_hook: ✅ Carryover completed:
:arrow_right_hook: 🛑 Blocked for now:
:arrow_right_hook: Pushed back:
:arrow_right_hook: Remaining:

⭐️ Points pulled in for this sprint: 14
⭐️✅ Points from this sprint completed:
⭐️🚫 Decided not to do:
⭐️🛑 Blocked for now:
⭐️ Remaining:

👾 Space invaders pulled in: 
👾✅ Space invaders completed:
👾 Remaining:

## Sprint activities

### Docket type #13978
1. ⭐️ (2) Intake| Docket Change | Receive VLJ ruling #15338  (Issue is Blocked)
1. ⭐️ (3) Intake| Docket Change| VLJ ruling view #15336
1. ⭐️ (2) Intake| Docket Change| Recommendation success banner #15333 

### Edit EP Claim Labels #11490
1. ⭐️(2) Edit EP Claim Labels | Non-rating to rating updates #15126
1. ⭐️(3) Edit EP Claim Labels | Ability to select new claim label #15112

**Comment by leikkisa** [comment]
Closing this ticket, we have transitioned to Jira.
